beacon_node/http_api/tests/broadcast_validation_tests.rs

```diff
@@ -7,16 +6,19 @@ use eth2::reqwest::StatusCode;
 use eth2::types::{BroadcastValidation, PublishBlockRequest};
 use http_api::test_utils::InteractiveTester;
 use http_api::{publish_blinded_block, publish_block, reconstruct_block, Config, ProvenancedBlock};
+use std::collections::HashSet;
 use std::sync::Arc;
 use types::{
-    BlobSidecar, Epoch, EthSpec, FixedBytesExtended, ForkName, Hash256, MainnetEthSpec, Slot,
+    ColumnIndex, Epoch, EthSpec, FixedBytesExtended, ForkName, Hash256, MainnetEthSpec, Slot,
 };
 use warp::Rejection;
 use warp_utils::reject::CustomBadRequest;
 
 type E = MainnetEthSpec;
 
 /*
+ * TODO(fulu): write PeerDAS equivalent tests for these.
```

Comment:
Note that I've updated some of the tests to support data columns, but we currently run them on Electra only. The tests that needs to be written for Fulu are the scenarios covering gossip blobs.